main/main.ino

```diff
@@ -984,6 +987,14 @@ void setup_wifi() {
 #  if defined(ESP32) && defined(ZgatewayBT)
     if (failure_number_ntwk > maxConnectionRetryWifi && lowpowermode)
       lowPowerESP32();
+#  elif defined(ESP32)
+    if (failure_number_ntwk > maxRetryWatchDog && !lowpowermode) {
+      watchdogReboot(2);
+    }
+#  else
+      if (failure_number_ntwk > maxRetryWatchDog) {
+        watchdogReboot(2);
+      }
```

Comment:
here is a proposal to cover the use of BT on ESP32:

```
#  if defined(ESP32)
    if (lowpowermode) {
      if (failure_number_ntwk > maxConnectionRetryWifi) {
        lowPowerESP32();
      }
    } else {
      if (failure_number_ntwk > maxRetryWatchDog) {
        watchdogReboot(2);
      }
    }
#  else
    if (failure_number_ntwk > maxRetryWatchDog) {
      watchdogReboot(2);
    }
#  endif
```

Reply:
Made one final tweak as lowPowerESP32() is part of the BT module, and changed `#  if defined(ESP32)` to `#  if defined(ESP32) && defined(ZgatewayBT)`

```
#  if defined(ESP32) && defined(ZgatewayBT)
    if (lowpowermode) {
      if (failure_number_ntwk > maxConnectionRetryWifi) {
        lowPowerESP32();
      }
    } else {
      if (failure_number_ntwk > maxRetryWatchDog) {
        watchdogReboot(2);
      }
    }
#  else
    if (failure_number_ntwk > maxRetryWatchDog) {
      watchdogReboot(2);
    }
#  endif
```

Reply:
that works!